components/developer/jenkins-swarm-client/files/jenkins-swarm-client.sample-conf

```diff
@@ -0,0 +1,147 @@
+# Example configuration for the jenkins-swarm-client service.
+# Technically this is a sourcable bash script which populates SWARM_CONFIG array.
+
+# Tune this for your deployment!
+JENKINS_MASTER_ROOT_URL="https://jenkins.localdomain"
+#JENKINS_PREFERRED_PROXY="http://proxy.localdomain:3128"
+
+# Container deployment script can add something else to desired labels
+# If a "$CUSTOM_LABELS_FILE.exclusive" exists, it is used (along with
+# a "$CUSTOM_LABELS_FILE" if present) with no generated additions.
+CUSTOM_LABELS_FILE="/opt/jenkins-swarm-client/etc/client.labels"
+
+# Avoid guessing the name and use one provided by container deployer
+# if this file exists and has one token inside.
+CUSTOM_WORKERNAME_FILE="/opt/jenkins-swarm-client/etc/client.workername"
+
+# No default => no option, if not specified
+CUSTOM_WORKERDESC_FILE="/opt/jenkins-swarm-client/etc/client.description"
+
+# Override some of the settings used for the swarm client
+CUSTOM_NUMEXECUTORS_FILE="/opt/jenkins-swarm-client/etc/client.numexecutors"
+DEFAULT_NUMEXECUTORS=1
+CUSTOM_WORKDIR_FILE="/opt/jenkins-swarm-client/etc/client.workdir"
+DEFAULT_WORKDIR="/tmp/jenkins-swarm-client"
+if [ -d /dev/shm ] && [ -w /dev/shm ] ; then
```

Comment:
This is a linux specific directory. Do we need this?

Reply:
In OI alone, probably not. The script originated as a multiplatform thing for various builders, and it is easier to maintain that way to set up swarm workers on different OSes with the same code.

Reply:
Well it would probably be better not to have too many complex configs. Otherwise it gets too confusing for new people. Let's keep configs we deliver via the OS group tailored to that OS. Everybody can have their own config afterwards.

Reply:
Hm, right. I first thought the question was about the script (which better be reusable). Sample config file indeed can be tailored.